lab-caleb/test/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+let assert = require('assert');
+const indexJS = require('../lib/index.js');
+
+console.log(indexJS.products);
```

Comment:
Knowing that your console.logging is great, but get rid of your logs before submission. We'll start docking for that next week.